src/ui/theme/theme.ts

```diff
@@ -121,4 +146,138 @@ export const theme = extendTheme({
     Menu: menuTheme,
     Link: linkTheme,
   },
+  textStyles: {
```

Comment:
Are these overrides to the chakra? If so:

1- Why use a number here instead of variables for values? There are for example letterSpacings variables on the system.

2- Do we need all of these overrides? Why not change what's needed and keep the feault value when it's not?

Reply:
1. Done. I am using variable names now
2. I am not overriding the defaults as discussed in our sync. I am just explicitly listing the defaults in our theme so that I can double check values within the code

Reply:
line-height should also be variables

Reply:
Coming up with names was difficult 😂, but it's done now